zephyr/module.yml

```diff
@@ -14,30 +14,30 @@ blobs:
     url: https://raw.githubusercontent.com/NXP/wifi_nxp/MCUX_2.15.100/wifi_bt_firmware/nw61x/uart_nw61x_se.h
     description: "Firmware for NXP IW612 Wi-Fi + Bluetooth single-chip solution"
     doc-url: https://www.nxp.com/products/wireless-connectivity/wi-fi-plus-bluetooth-plus-802-15-4/2-4-5-ghz-dual-band-1x1-wi-fi-6-802-11ax-plus-bluetooth-5-4-plus-802-15-4-tri-radio-solution:IW612
-  - path: rw61x/rw61x_sb_ble_15d4_combo_a2.bin
-    sha256: 7d94d41c4778b691292a325b881e972cb676467e57bfc804b0d6f8926d5a8dda
+  - path: rw61x/rw61x_sb_combo_a2.bin
```

Comment:
let's keep it `rw61x_sb_ble_15d4_combo_a2.bin` here, the name used was fixed here: https://github.com/zephyrproject-rtos/hal_nxp/commit/ee22de2eb4adf53a43d29dc79f4493a5ee154189

If you change change back this line, I think west will renamed automatically the file, and we'll be good

Reply:
update, please review again